zc_install/sql/updates/mysql_upgrade_zencart_158.sql

```diff
@@ -36,6 +36,7 @@ TRUNCATE TABLE db_cache;
 
 
 ALTER TABLE layout_boxes ADD plugin_details varchar(100) NOT NULL default '';
+ALTER TABLE manufacturers ADD COLUMN featured tinyint default 0;
```

Comment:
Just a question:  Will this cause issues if I backport this to a zc157 installation when the site's upgraded to v158?

Reply:
While that line will fail, zc_install "should" just note it in the `upgrade_exceptions` table and carry on.  Of course, we should test that. I don't recall off the top of my head whether `ADD` vs `ADD COLUMN` is the specific syntax it checks for (or both).